src/test/java/teammates/test/cases/ui/browsertests/FeedbackRubricQuestionUiTest.java

```diff
@@ -73,22 +73,29 @@ private void testInstructorResultsPage() {
 
         // Question view
         instructorResultsPage = loginToInstructorFeedbackResultsPageWithViewType("teammates.test.instructor", "openSession2", false, "question");
+        instructorResultsPage.waitForPanelsToCollapse();
+        
         instructorResultsPage.verifyHtmlMainContent("/instructorFeedbackResultsPageRubricQuestionView.html");
         
+        
         // Giver Recipient Question View
         instructorResultsPage = loginToInstructorFeedbackResultsPageWithViewType("teammates.test.instructor", "openSession2", false, "giver-recipient-question");
+        instructorResultsPage.waitForPanelsToCollapse();
```

Comment:
Yes, the 'not submitted' table is shared across the the different views, so they're now loaded by ajax too.